app/src/main/java/org/systers/mentorship/remote/datamanager/RelationDataManager.kt

```diff
@@ -11,67 +10,61 @@ import org.systers.mentorship.remote.responses.CustomResponse
  */
 class RelationDataManager {
 
-    private val apiManager = ApiManager.instance
+    private val relationService = ApiManager.instance.relationService
```

Comment:
Why did you removed the **ApiManager** instance from here?

Reply:
as it would shorten the code. if you take a closer look at the functions in this file, they all use apiManager.relationService and it's always great to avoid the same code which could be shortened :)

Reply:
It will also improve the code readability

Reply:
Actually, all the data managers are using **ApiManager** instance to access the api calls, therefore, I think you should not remove this from here because if you would do that from here, all the data managers class also needs to be changed and we need to discuss that from the maintainer of the project before doing this type of change. 
So it's better to add **ApiManager** instance here also, at this moment. Hope you understand :)

Reply:
well, I understand your point, but I haven't removed it. 
`ApiManager.instance.relationService`
I just directly use it to call the relation service. 
it's like in math when you have something like 8x + 16x - 4x = 0 which equals to 4(2x + 4x x) and that could be shortened too. But I didn't remove the 4 out of the equation. I just shortened it. It's not a serious change that I made.

Reply:
And the code works the same way it worked before. Do I still need to bring it back? :)

Reply:
Yeah, I understand your point. Okay no problem, you don't have to bring that back

Reply:
Thanks!